src/app/tools/import.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="page-header">
     <h1>{{'importData' | i18n}}</h1>
 </div>
-<form #form (ngSubmit)="submit()" [appApiAction]="formPromise" ngNativeValidate>
```

Comment:
does removing the `formPromise` default behavior of the `appApiAction` also remove the spinner/loading behavior as well? If so we'll need to replace that and ensure it's working.